packages/jest-jasmine2/package.json

```diff
@@ -18,19 +18,18 @@
   },
   "dependencies": {
     "@jest/environment": "^28.0.0-alpha.1",
+    "@jest/expect": "^28.0.0-alpha.1",
     "@jest/source-map": "^28.0.0-alpha.0",
     "@jest/test-result": "^28.0.0-alpha.1",
     "@jest/types": "^28.0.0-alpha.1",
     "@types/node": "*",
     "chalk": "^4.0.0",
     "co": "^4.6.0",
-    "expect": "^28.0.0-alpha.1",
     "is-generator-fn": "^2.0.0",
     "jest-each": "^28.0.0-alpha.1",
     "jest-matcher-utils": "^28.0.0-alpha.1",
     "jest-message-util": "^28.0.0-alpha.1",
     "jest-runtime": "^28.0.0-alpha.1",
-    "jest-snapshot": "^28.0.0-alpha.1",
```

Comment:
can we remove the direct dependency from circus as well?

Reply:
Ups.. It was necessary here: 

https://github.com/facebook/jest/blob/199f9811ae68b15879cbe18b7ef7ebd61eefcf23/packages/jest-jasmine2/src/setup_jest_globals.ts#L10-L14